nss-evening-cohort-26/pos-terminal-the-algorithm-avengers
# Styling and back to order btn

## Description
- Added styling to overall app. 
- The login pg with big logo and background image still needs work though 
- added the btn that allows user to go back to order in the middle on the show items not in order page

## Related Issue


## Motivation and Context


## How Can This Be Tested?
Everything should be working and when you click add item to order a btn should be under the items not in order that takes you back to the order you were adding items to. 

## Screenshots (if appropriate):

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
